alephant-renderer.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake-rspec"
 
   spec.add_runtime_dependency 'i18n'
-  spec.add_runtime_dependency 'mustache', '>= 0.99.5'
+  spec.add_runtime_dependency 'mustache', '~> 0.99.5'
```

Comment:
@bbc-fossbot Haven't got the luxury of time I'm afraid, and now stops gem installing from app consuming this gem upstream.